stories/documentation/feedback/callout/callout-tiny.stories.ts

```diff
@@ -1,70 +1,52 @@
-import { Meta, Story } from '@storybook/angular';
-
-interface CalloutTinyStory {
-	s: boolean;
-	palette: string;
-	icon: string;
-}
+import { Meta, StoryObj } from '@storybook/angular';
+import { CalloutComponent } from '@lucca-front/ng/callout';
+import { default as BasicStory } from './callout-basic.stories';
 
 export default {
-	title: 'Documentation/Feedback/Callout/Tiny',
+	...BasicStory,
+	title: 'Documentation/Feedback/Callout/Tiny'
+} as Meta;
+
+export const Template: StoryObj<CalloutComponent & { description: string }> = {
```

Comment:
I wonder if we still have to use separate files for stories. It would be easier to share some part if they were in the same file.

@jeremie-lucca does ZeroHeight support importing one story from a file containing multiple one?

Reply:
Because we won't have an answer until at least two weeks iirc, shouldn't we close this comment and tackle this in another future PR?